django/views/debug.py

```diff
@@ -71,6 +71,14 @@ def get_safe_settings():
     return settings_dict
 
 
+def get_safe_request_meta(meta):
+    "Returns a dictionary of the request.META, with sensitive request.META blurred out."
+    new_meta = {}
+    for k, v in meta.items():
+        new_meta[k] = cleanse_setting(k, v)
+    return new_meta
```

Comment:
This could be simplified as `{k: cleanse_setting(k, v) for k, v in meta.items()}`.